docs/CppInteroperability/GettingStartedWithCppInterop.md

```diff
@@ -0,0 +1,209 @@
+#  Getting started with C++ Interoperability
+
+This document is desgined to get you started with bidirectional API-level interoperability between Swift and C++.
+
+## Table of Contents
+
+- [Creating a Module to contain your C++ source code](#creating-a-module-to-contain-your-c-source-code)
+- [Adding C++ to an Xcode project](#adding-c-to-an-xcode-project)
+- [Creating a Swift Package](#Creating-a-Swift-Package)
+- [Building with CMake](#building-with-cmake)
+
+## Creating a Module to contain your C++ source code
+
+- Create a new C++ implementation and header file, and generate a bridging header when prompted
+- For this example we will call the files Cxx, so we should have a Cxx.cpp and Cxx.hpp.
+- Import your C++ header, into the bridging header 
```

Comment:
I think there might be a missunderstanding here. There are two paths for importing C++ headers, you can either use a bridging header _OR_ a module. 

If you use a bridging header, you can tell the compiler where to find it via `-import-objc-header`. Sometimes bridging headers work better/are easier because they don't require modularization. It's more similar to how C++ has traditionally imported things. 

Modules are much preferred, though. They allow headers to be grouped and specific modules to be imported. It's more clear to the user what APIs they are importing/using. It also allows for much better compile times (faster lookups, lazy lookups, etc.) and improved incremental builds. Because the model matches Swift, it also makes some features work better, such as extensions. You don't need any flags to use a module, you can just set the include path, and the modules will be picked up if there's a module map.

Reply:
ahh okay, i will remove that for now and add more detail later